cmd/soroban-cli/src/secure_store.rs

```diff
@@ -0,0 +1,64 @@
+use sep5::SeedPhrase;
+use stellar_strkey::ed25519::PrivateKey;
+
+use crate::{
+    config::{address::KeyName, locator, secret::{self, Secret}}, print::Print, signer::keyring::{self, StellarEntry}
+};
+
+pub struct SecureStore {}
```

Comment:
not sure if this needs its own mod, but wanted to pull the code out where both `add` and `generate` could use it